include/TrustWalletCore/TWStellarPassphrase.h

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+#include "TWBase.h"
+
+TW_EXTERN_C_BEGIN
+
+TW_EXPORT_ENUM()
+enum TWStellarPassphrase {
+    TWStellarPassphraseStellar     /* "Public Global Stellar Network ; September 2015" */,
+    TWStellarPassphraseKin         /* "Kin Mainnet ; December 2018" */,
+};
+
+static const char *_Nonnull TWStellarPassphrase_Stellar = "Public Global Stellar Network ; September 2015";
+static const char *_Nonnull TWStellarPassphrase_Kin = "Kin Mainnet ; December 2018";
```

Comment:
Let's use something more than an underscore to distinguish from enum cases. Maybe ` TWStellarPassphraseStellarString`

Reply:
This is just an auto-generated file that got merged in with the latest changes.

The file is this way at HEAD: https://github.com/TrustWallet/wallet-core/blob/d8d408c33085b9466ad007ed45f880ee43720d4e/include/TrustWalletCore/TWStellarPassphrase.h